openapi/components/schemas/GatewayAccountConfig/CheckoutCom.yaml

```
@@ -23,5 +26,8 @@ allOf:
             type: boolean
             description: Specifies whether to mark pending payouts as 'waiting-gateway/unknown' instead of 'approved'.
             default: false
+          subEntityIdWebsiteCustomField:
+            type: string
+            description: Name of website custom field containing sub-entity ID.
```

suggestion: to avoid [noun stacks](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
            description: Name of the website custom field that contains the sub-entity ID.
```